tests/acceptance/index-test.js

```diff
@@ -25,12 +25,14 @@ test('visiting /', assert => {
 
   andThen(function() {
     assert.equal(currentURL(), '/?a=third');
+    assert.ok($('[data-a=third]').hasClass('highlight'), 'Element is highlighted');
```

Comment:
I'm not particularly pleased with the approach of testing the feature in an acceptance test, but it does the job for now at least.